packages/lesswrong/server/scripts/exportPostDetails.ts

```diff
@@ -92,7 +97,16 @@ Vulcan.exportPostDetails = wrapVulcanAsyncScript(
     await fs.writeFile(filePath, csvFile)
     //eslint-disable-next-line no-console
     console.log(`Wrote details for ${rows.length} posts to ${filePath}`)
+  }
+)
+
+Vulcan.exportLowKarma = ({outputFilepath, karma = 5}: {outputFilepath: string, karma?: number}) => {
```

Comment:
Should the `5` magic number be set as a `const` at the top of the file? Then you can `export/import` that into the other migration.